WORKSPACE

```diff
@@ -186,11 +186,6 @@ stack_snapshot(
     local_snapshot = "//:ghcide-stack-snapshot.yaml",
     packages = [
         "ghcide",
-        # Needed by data-default-instances-containers and data-default-instances-old-locale
-        "base",
-        "containers",
-        "data-default-class",
-        "old-locale",
```

Comment:
Yes, it does test the feature. Going forward it would be preferable to have dedicated tests for `vendored_packages` in general. I've opened https://github.com/tweag/rules_haskell/issues/1587 to track this.